modules/helfi_tpr_config/helfi_tpr_config.install

```diff
@@ -202,3 +202,12 @@ function helfi_tpr_config_update_9036(): void {
   \Drupal::service('helfi_platform_config.config_update_helper')
     ->update('helfi_tpr_config');
 }
+
+/**
+ * UHF-8479 Update standard page & landing page to include
+ * service_list_search paragraph.
+ */
+function helfi_tpr_config_update_9037(): void {
+  \Drupal::service('helfi_platform_config.config_update_helper')
```

Comment:
This update hook will trigger a lot of customisation reverts in instances. As you only want to add enable a paragraph type, you can use the `helfi_platform_config_update_paragraph_target_types()` function to achieve the same change but without the customisation reverts.

What to do? 
Change this: 
```
\Drupal::service('helfi_platform_config.config_update_helper')->update('helfi_tpr_config');
``` 
to this:
```
helfi_platform_config_update_paragraph_target_types();
```